deploy/kubernetes/postee.yaml

```diff
@@ -187,7 +194,9 @@ spec:
             - name: postee-config
               mountPath: /config
             - name: rego-template
-              mountPath: /server/rego-template/custom
+              mountPath: /server/rego-template
+            - name: rego-filters
+              mountPath: /server/rego-filters
```

Comment:
@krol3 
Can users add something custom to this folder?

Reply:
If I understand correctly, the user could add more rego samples  https://github.com/aquasecurity/postee/tree/main/rego-filters

Reply:
Hi @ krol3
We have a mistake: postee is using the rego-template**s** folder.
In addition, if we mount Kubernetes volumes `/server/rego-templates` and `/server/rego-filters`, then these volumes will be without rules by default, which we copy from the docker image. It is required to mount the folders for rego samples in subfolders, for example server `/rego-templates/custom`, after which the user can save and use their rego samples.

Reply:
@DmitriyLewen please take a look to the folders

Reply:
Thanks @DmitriyLewen  I applied your changes.

Reply:
@AndreyLevchenko  could you review this PR? what is missing to merge it?